src/lib/api/base.js

```diff
@@ -11,7 +11,8 @@ const apiConfig = {
 };
 const http = axios.create(apiConfig);
 
-const HTTP_STATUS_CODES_WITH_ERROR_PAGE = [404, 429, 500];
+const HTTP_STATUS_CODES_WITH_ERROR_PAGE = [404, 410, 429, 500];
+const HTTP_STATUS_CODES_WITHOUT_NOTIFICATION = [410];
```

Comment:
I am wondering if all the error codes above should not show a notification, given that they will already redirect to a dedicated error page. WDYT @kpsherva ?

Reply:
if I remember correctly, not all the codes have the error page, that was the original bug in the first place - and I agree, the ones which have error page, should not show the notification by default.
In this case the solution would be to implement those pages, instead of adding additional condition (without notification)

Reply:
So the code lists here should be for showing error page and for notifications (and not without as it currently is here) separately?
That way the existing pages won't have the notification anymore and we can configure codes where we just want to show a notification by just adding the code in the notification list.